### PR TITLE
[WNMGDS-570] Update installation, css, font image documentation

### PR DESCRIPTION
**Pull request by bernardwang**
### Summary
- Updates "Install with NPM" page to be "Installation"
- Merge "Using font and images" with the "Using Sass/CSS" page
- Add additional font and image path examples 
- Move browser support page to guidelines

### How to test
- Run the doc site and ensure all the content and links are working
- Review the content

### CODE REVIEW

File: packages/design-system-docs/src/pages/startup/installation.md [hunk 1]
```diff
@@ -1,26 +1,77 @@
 ---
-title: Install with NPM
+title: Installation
 weight: 0
 ---
 
-The design system is available an NPM package or via a <a href="{{root}}/download.zip">.zip download</a>.
+How you implement the design system depends on the needs of your project and your workstyle. We highly recommend implementing the design system with `npm`, as allows you to easily update the design system version, but we also provide a direct download if `npm` will not work for you or your project.
```
Comment (line47) [inline packages/design-system-docs/src/pages/startup/installation.md:6]:
```suggestion
We highly recommend implementing the design system with `npm`, as it allows 
```

File: packages/design-system-docs/src/pages/startup/installation.md [hunk 2]
```diff
@@ -1,26 +1,77 @@
...
+```
+
+## Download zip
+
+<a href="{{root}}/download.zip">Download the zip file</a> from the latest release and extract the zip file to see the following folder structure. Like the `npm` package, the contents of `download/design-system/dist` contain design system code and assets.
 
 ```
-npm install --save {{npm}}
+└── download
+    ├── design-assets/      Design system assets (Sketch files, designer assets)
```
Comment (line47) [inline packages/design-system-docs/src/pages/startup/installation.md:59]:
```suggestion
    ├── design-assets/      Design system assets (Sketch files, fonts, designer assets)
```

File: packages/design-system-docs/src/pages/startup/sass-and-css.md [hunk 3]
```diff
@@ -3,38 +3,95 @@ title: Using Sass/CSS
...
+
+```
+SASS_PATH="node_modules"
+```
 
-[View an example](https://github.com/CMSgov/design-system/blob/master/examples/react-app/src/styles/index.scss)
+If you have your own custom Webpack configuration, you need to use [`file-loader`](https://webpack.js.org/loaders/file-loader/) and [`resolve-url-loader`](https://github.com/bholloway/resolve-url-loader) to copy assets into your build folder and inject the correct URLS to your build files.
 
-<h4 class="ds-h4 ds-u-font-size--base">Applying styles to your page</h4>
+## Applying styles to your page
```
Comment (line47) [inline packages/design-system-docs/src/pages/startup/sass-and-css.md:94]: the `utility class` link below should be updated to `{{root}}/utilities/overview/`

File: packages/design-system-docs/src/pages/startup/sass-and-css.md [hunk 4]
```diff
@@ -3,38 +3,95 @@ title: Using Sass/CSS
...
+Without overriding these variables, it's possible that your fonts and images will not render correctly in your project.
+
+#### Setting asset paths to `node_modules`
+
+If your project uses Webpack or Create React App, you can set the asset paths directly to the fonts and image folders in your `node_modules`.
+
+```css
+$font-path: '~{{npm}}/dist/fonts';
+$image-path: '~{{npm}}/dist/images';
+```
```
Comment (nichia) [inline packages/design-system-docs/src/pages/startup/sass-and-css.md:82]: Which file do is this added to?

File: packages/design-system-docs/src/pages/startup/theming.md [hunk 5]
```diff
@@ -62,6 +64,40 @@ set the `$ds-include-focus-styles` variable to `true`.
...
+    box-shadow: 0 0 0 3px $focus-text-field-color-highlight, inset 0 0 0 1px $focus-text-field-color;
+    border-color: $focus-text-field-color;
+  }
+  font-size: $h4-font-size;
+  border-radius: 5px;
+  border: 2px solid $custom-color;
+}
+```
+
+Unlike Sass variable overrides, these styles must be declared _after_ importing the design system. These are often used in conjuntion with Sass variable overrides.
```
Comment (nichia) [inline packages/design-system-docs/src/pages/startup/theming.md:84]: missing `c` in `conjunction`

File: packages/design-system-docs/src/pages/startup/sass-and-css.md (hunk 4 above)
Comment (bernardwang) [reply packages/design-system-docs/src/pages/startup/sass-and-css.md:82]: Do you mean where is the font and image paths defined? It just has to happen in a CSS file before you import the design system. I recommend looking through the child design system example or any of the hc.gov or m.gov child design systems

**Comment by bernardwang** [comment]
@line47 @nichia I updated the child design system documentation as well.

File: packages/design-system-docs/src/pages/startup/installation.md (hunk 1 above)
Comment (line47) [reply packages/design-system-docs/src/pages/startup/installation.md:6]: I think adding this `it` is still needed

File: packages/design-system-docs/src/pages/startup/sass-and-css.md (hunk 4 above)
Comment (bernardwang) [reply packages/design-system-docs/src/pages/startup/sass-and-css.md:82]: https://github.com/CMSgov/design-system/pull/763/files#diff-c637d2d90071b64e35d7490418de01f2R86

File: packages/design-system-docs/src/pages/startup/child-design-systems.md [hunk 6]
```diff
@@ -3,31 +3,36 @@ title: Child Design Systems
...
-
-### Child design system guidelines
-
-- A child design system should be actively maintained by product teams using the child design system. The core CMSDS team is not responsible for updating child design systems, however we are responsible for guidelines and assisting with training and adoption.
-- Users of child design system should contribute back to the larger CMS design system community by filing issues/bugs in Github or Jira, giving feedback, and general maintenance. Collaboration and participation are essential to the success of the CMS design system family.
-- A child design system should defer to the core CMS design system for best practices or accessibility guidelines.
+**Note:** Child design systems were previously called "site packages". This name change reflects our vision for these systems to be more independent and self sustaining within the CMS Design System family. Many of the internal changes in the v2 release aim to facilitiate this vision by simplifying the experience of maintaining and contributing to child design systems.
```
Comment (nichia) [inline packages/design-system-docs/src/pages/startup/child-design-systems.md:19]: Typo - `facilitate`

**Pull request merged**